scrapy/utils/defer.py

```diff
@@ -113,3 +117,27 @@ def iter_errback(iterable, errback, *a, **kw):
             break
         except Exception:
             errback(failure.Failure(), *a, **kw)
+
+
+def isfuture(o):
```

Comment:
I hope this isn't too much bikeshedding, but I wonder if this function shouldn't be underscored, if it's only needed for py35